async/ratatui-counter/rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "stable"
+channel = "nightly"
```

Comment:
This is only needed for formatting.

Reply:
I'd suggest turning those settings off instead. At least for a template that we ship.

Reply:
Yup I agree

Reply:
Done!

<img width="477" alt="image" src="https://github.com/ratatui-org/templates/assets/1813121/71a92e4e-bcae-47c1-82bc-a7f1c2ceb870">

<img width="538" alt="image" src="https://github.com/ratatui-org/templates/assets/1813121/b7d88e49-98af-4583-a213-a10a6b362f4a">